flocker/control/script.py

```diff
@@ -30,6 +30,10 @@ class ControlOptions(Options):
          "The external API port to listen on."],
         ["agent-port", "a", 'tcp:4524',
          "The port convergence agents will connect to."],
+        ["certificate-path", "c", None,
+         ("Absolute path to directory containing the cluster "
+          "root certificate and control service certificate "
+          "and private key. Defaults to /etc/flocker")],
```

Comment:
`twisted.python.usage` supports defaults, which has benefit of simplifying code slightly but also means you don't need to duplicate "/etc/flocker" in the help string.